server/server/src/__tests__/server.test.ts

```diff
@@ -115,7 +115,7 @@ describe('server', () => {
   })
 
   test('should connect to server', async () => {
-    expect(conn.readyState).toEqual(1)
+    expect(await Promise.resolve(conn.readyState)).toEqual(WebSocket.OPEN)
```

Comment:
Why we need a promise here?

Reply:
Because async requires await.

Reply:
I believe we need to remove `async` here

Reply:
@haiodo , @z2sx  Thanks, fixed!